src/apps/api/serializers/competitions.py

```diff
@@ -98,6 +100,15 @@ def validate_leaderboards(self, value):
             raise serializers.ValidationError("Competitions require at least 1 leaderboard")
         return value
 
+    def validate_phases(self, phases):
+        if not phases or len(phases) <= 0:
+            raise serializers.ValidationError("Competitions must have at least one phase")
+        if phases[0]['auto_migrate_to_this_phase'] is True:
+            raise serializers.ValidationError("You cannot auto migrate to the first phase of a competition")
+        if len(phases) == 1 and phases[0]['auto_migrate_to_this_phase']:
```

Comment:
This validation step is unreachable. Need to move it above checking just `phase[0]` in order for it to trigger.

Reply:
Made code reachable in latest push